controller/service.go

```diff
@@ -12,24 +12,40 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+const (
+	birthdayDatabase        = "birthdayDB"
+	birthdayCollection      = "birthdays"
+	personalNumberParameter = "personalNumber"
+	dateParameter           = "date"
+	nameParameter           = "name"
```

Comment:
Better used with first letters as upper-case and with a short description, but accepted.

Reply:
ok..
done